Artesian/Artesian.SDK/Factory/VersionedTimeSerie.cs

```diff
@@ -164,5 +166,37 @@ public async Task Save(Instant downloadedAt, bool deferCommandExecution = false,
             //else
             //    _logger.Warn("No Data to be saved.");
         }
+
+        /// <summary>
+        /// MarketData Delete
+        /// </summary>
+        /// <remarks>
+        /// Delete the Data of the current MarketData
+        /// </remarks>
+        /// <param name="rangeStart">LocalDateTime start of range to be deleted</param>
+        /// <param name="rangeEnd">LocalDateTime end of range to be deleted</param>
+        /// <param name="version">Version of the Time Serie</param>
+        /// <param name="product">Product of the MarketAssessment, BidAsk or Auction Time Serie</param>
+        /// <param name="deferCommandExecution">DeferCommandExecution</param>
+        /// <param name="deferDataGeneration">DeferDataGeneration</param>
+        /// <param name="ctk">The Cancellation Token</param> 
+        /// <returns></returns>
+        public async Task Delete(LocalDateTime rangeStart, LocalDateTime rangeEnd, LocalDateTime? version = null, List<string> product = null, bool deferCommandExecution = false, bool deferDataGeneration = true, CancellationToken ctk = default)
```

Comment:
This is wrong. This is a VersionedTimeSerie representing a Single Selected Version. This Delete should only take an optional dataRange (otherwise, delete whole). Version and Product are implicit.